doc/tutorials/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst

```diff
@@ -165,11 +165,11 @@ Step 4: Use Motion Planning with the Panda
 
   * Make sure both states are not in collision with the robot itself.
 
-  * Make sure the Planned Path is being visualized. Also check the
-    **Show Trail** checkbox in the **Planned Path** tree menu.
+  * Un-check the **Show Trail** checkbox in the **Planned Path** tree menu.
```

Comment:
I noticed I had to uncheck and recheck the box to get RViz to show the trail, so to minimize confusion I started with the box unchecked and told the user to check the box after making the plan